src/nyc_trees/js/src/group_list.js

```diff
@@ -1,4 +1,41 @@
 "use strict";
 
+var $ = require('jquery');
+
+var attrs = {name: 'data-group-name',
+             description: 'data-group-description'},
+    dom = {searchInput: '[data-class="group-search"]',
```

Comment:
I think it's a best practice to only use `id` and `data-*` attributes with behaviors and never `class` to keep interop with designers smooth. I like to take this further and avoid using `id` when performance is not a consideration. I'm not sure if the latter is advisable.